public/js/gulliver.es6.js

```diff
@@ -85,12 +85,14 @@ class Gulliver {
       subtitle: true
     });
 
-    const setupLighthouseChart = () => {
-      new LighthouseChart().load();
+    const setupCharts = () => {
+      new Charts('lighthouse-chart', '/api/lighthouse/graph/').load();
+      new Charts('psi-chart', 'https://web-performance-dot-pwa-directory.appspot.com/pagespeedreport/').load();
+      new Charts('wpt-chart', 'https://web-performance-dot-pwa-directory.appspot.com/webpagetestreport/').load();
```

Comment:
Suggestion: change this to

```
const chartConfigurations = [
  {id: 'lighthouse-chart', url: '/api/lighthouse/graph/'},
  {id: 'psi-chart', url: 'https://web-performance-dot-pwa-directory.appspot.com/pagespeedreport/'},
  {id: 'wpt-chart', url: 'https://web-performance-dot-pwa-directory.appspot.com/webpagetestreport/'},
];

chartConfigurations.forEach(config => new Chart(config).load())
```

This makes it easier to change the chart API and doesn't repeat the initialization logic. This requires also changing the chart constructor to accept a config object.